src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -109,6 +109,8 @@ public void snapshots(UIThreadSet<Activity> liveActivities, OutputStream out) th
             if (MPConfig.DEBUG) {
                 Log.i(LOGTAG, "Screenshot took more than 1 second to be scheduled and executed. No screenshot will be sent.", e);
             }
+            writer.write("\"\"");
```

Comment:
I'm not sure this is actually what we want to return in this case. What does the output look like here? Should this be the empty list rather than an empty string?